cudaaligner/CMakeLists.txt

```diff
@@ -13,9 +13,11 @@ project(cudaaligner)
 # Project specific NVCC flags
 if((CUDA_VERSION_MAJOR GREATER 10) OR (CUDA_VERSION_MAJOR EQUAL 10 AND CUDA_VERSION_MINOR GREATER 0))
 set(CUDA_NVCC_FLAGS "${CUDA_NVCC_FLAGS} -std=c++14 --expt-relaxed-constexpr")
+
```

Comment:
@atadkase can you remove this extra line here? I'm approving conditionally so once the tests have been run again after merge with master we can push it

Reply:
Will do

Reply:
Done.